src/containers/edit-profile/profile-tab/profile-tab-inner-container/ProfileTabInnerContainer.tsx

```diff
@@ -20,6 +20,11 @@ const ProfileTabInnerContainer: FC<ProfileTabProps> = ({ user }) => {
     profileData.generalData.errors &&
     Object.values(profileData.generalData.errors).some((error) => error)
 
+  const handleUpdateProfile = () => {
+    handleSubmit(profileData.generalData.data)
+    window.location.reload()

```

Comment:
I guess it's better to move this into the `handleResponse` function in the `useUpdateUser` hook. Because we need to reload only if the user info was updated successfully.

Also, do we need to reload the page if only text fields were changed?

One last thought: I think we should either add a short delay before reloading to give users a chance to read the message from the snackbar, or get rid of it completely. Because right now we don't have enough time to read it, and it's a bit confusing.

Reply:
Thanks for comment. Fixed! Now page reload only if photo changed and after short delay.